client/src/App.js

```diff
@@ -17,6 +16,7 @@ function App() {
 			<Router>
 				<Switch>
 					<Route path="/" exact component={Home} />
+					<Route path= "/api/home/success-stories" exact component={Home} />
```

Comment:
what is this!!

Reply:
ah don't worry, I find this after pushing the request so I updated it but didn't updated her yet.